website/language.py

```diff
@@ -72,6 +72,21 @@
 
 INVALID_EMAIL_CONFIRM_TOKEN = 'This confirmation link is invalid. Please <a href="/login/">log in</a> to continue.'
 
+CANNOT_MERGE_ACCOUNTS_SHORT = 'Cannot Merge Accounts'
+
+CANNOT_MERGE_ACCOUNTS_LONG = 'Accounts cannot be merged due to a conflict.'
```

Comment:
Though I agree with that approach -- Almost a guarantee that QA will want more direction to the user! Can we list *possible* conflicts w/direction, without building into the code the auto-delivery of the exact conflict?